misc/config_tools/configurator/README.md

```diff
@@ -78,11 +79,11 @@ Run following command in the 'acrn-hypervisor' directory.
 ```shell
 cd misc\config_tools
 python scenario_config/schema_slicer.py
-python scenario_config/jsonschema/convert.py
+python scenario_config/jsonschema/converter.py
```

Comment:
Don't we need backslashes here? I have run the command in the Anaconda Prompt on my Windows PC, and I need this command: `python scenario_config\jsonschema\converter.py`
Maybe it's different if you use the PowerShell (I haven't tried that). But the document does not instruct users to use the PowerShell.